crates/vm/levm/src/opcode_handlers/environment.rs

```diff
@@ -32,12 +32,13 @@ impl<'a> VM<'a> {
         let (account, address_was_cold) = self
             .db
             .access_account(&mut self.accrued_substate, address)?;
+        let account_info = account.info.clone();
```

Comment:
```suggestion
        let balance = account.info.balance;
```
We can do this and then just push the balance instead of cloning account info. Performance is mostly the same cause AccountInfo is still small but this is just nicer.